homeassistant/components/togrill/sensor.py

```diff
@@ -122,6 +122,8 @@ def available(self) -> bool:
     @property
     def native_value(self) -> StateType:
         """Get current value."""
-        if packet := self.coordinator.data.get(self.entity_description.packet_type):
+        if packet := self.coordinator.data.get(
+            (self.entity_description.packet_type, None)
+        ):
```

Comment:
why do we have this?

Reply:
I suppose i could have typed it as Packet | tuple[Packet, int]. But feels cleaner to not mix.